Dockerfile

```diff
@@ -27,21 +27,26 @@ RUN dpkg --add-architecture armel \
         libnl-3-dev \
         libprotobuf-c-dev \
         libprotobuf-dev \
-        libseccomp-dev \
-        libseccomp-dev:arm64 \
-        libseccomp-dev:armel \
-        libseccomp-dev:armhf \
-        libseccomp-dev:ppc64el \
-        libseccomp2 \
         pkg-config \
         protobuf-c-compiler \
         protobuf-compiler \
         python-minimal \
         sudo \
         uidmap \
+	astyle \
+	golint \
+	gperf \
     && apt-get clean \
     && rm -rf /var/cache/apt /var/lib/apt/lists/*;
 
+RUN wget https://github.com/seccomp/libseccomp/releases/download/v2.5.0/libseccomp-2.5.0.tar.gz \
```

Comment:
Can we `ADD` a helper script?

Reply:
Sure. Done as part of https://github.com/opencontainers/runc/pull/2682/commits/f549f8989c9db32dec19ee6c4ad9bce5fa5abf32.